azure/datalake/store/lib.py

```diff
@@ -70,9 +75,9 @@ def refresh_token(token):
 
 
 def auth(tenant_id=default_tenant, username=default_username,
-         password=default_password, client_id=default_client,
+         password=None, client_id=default_client,
```

Comment:
This is better, I think this applies to username, tenant, username, secret as well.